kong/resty/dns/client.lua

```diff
@@ -714,6 +713,13 @@ local function individualQuery(qname, r_opts, try_list)
 
   local result
   result, err = r:query(qname, r_opts)
+  -- Manually destroy the resolver.
+  -- When resovler is initialized, some socket resources are also created inside
+  -- resolver. As the resolver is created in timer-ng, the socket resources are
```

Comment:
There is no timer-ng in 2.8.x.

Reply:
good catch

Reply:
I think it has been resolved in https://github.com/Kong/kong/pull/12698/commits/81e19b2ffd4ec939db15291723e8931b7d1b5449, so resolve this conversation